README.md

```diff
@@ -4,16 +4,150 @@ AtacWorks is a deep learning toolkit for track denoising and peak calling from l
 
 ![AtacWorks](data/readme/atacworks_slides.gif)
 
-AtacWorks trains a deep neural network to learn a mapping between noisy (low coverage/low quality) ATAC-Seq data and matching clean (high coverage/high quality) ATAC-Seq data from the same cell type. Once this mapping is learned, the trained model can be applied to improve other noisy ATAC-Seq datasets. 
 
-AtacWorks models can be trained using one or more pairs of matching ATAC-Seq datasets from the same cell type. AtacWorks requires three specific inputs for each such pair of datasets:
-1. A coverage track representing the number of sequencing reads mapped to each position on the genome in the low-quality dataset.
-2. A coverage track representing the number of sequencing reads mapped to each position on the genome in the high-quality dataset. 
-3. The genomic positions of peaks called on the high-quality dataset. These can be obtained by using MACS2 or any other peak caller.
-The model learns a mapping from (1) to both (2) and (3); in other words, from the noisy coverage track, it learns to predict both the clean coverage track, and the positions of peaks in the clean dataset. We also provide pretrained models that can be applied to a noisy dataset.
+## System Setup
+
+### System requirements
+
+* Ubuntu 16.04+
+* CUDA 9.0+
+* Python 3.6.7+
+* GCC 5+
+* (Optional) A conda or virtualenv setup
+* Any NVIDIA GPU. AtacWorks training and inference currently does not run on CPU.
+
+### Install dependencies
+
+* Download `bedGraphToBigWig` and `bigWigToBedGraph` binaries and add to your $PATH
+    ```
+    rsync -aP rsync://hgdownload.soe.ucsc.edu/genome/admin/exe/linux.x86_64/bedGraphToBigWig <custom_path>
+    rsync -aP rsync://hgdownload.soe.ucsc.edu/genome/admin/exe/linux.x86_64/bigWigToBedGraph <custom_path>
+    export PATH="$PATH:<custom_path>"
+    ```
+
+* Install pip dependencies
+
+    ```
+    pip install -r requirements-pip.txt
+    ```
+## Sample installation test
+* Run this command to ensure your environment is setup correctly.
```

Comment:
What command?

Reply:
still haven't decided, so left it blank.